protocols_utils/code_generation/avrotools.py

```diff
@@ -241,21 +242,21 @@ def handleBytes(self, datum):
             return self.sinkValue
 
     def handleInt(self, datum):
-        if not ((isinstance(datum, int) or isinstance(datum, long)) and
+        if not ((isinstance(datum, int)) and
```

Comment:
are we sure `long` is not needed here?

Reply:
This is nearly right, but needs and additional import similar to the above. I will correct this.

Reply:
@kevinpetersavage not sure if this fixed, waiting your feedback before accepting